src/components/Input/index.js

```diff
@@ -3,8 +3,8 @@ import './style.css';
 export default function Input({name, disabled=false, textholder}){
     return (
         <div className="input">
-            <label id="form-label" htmlFor="form-input">{name}</label>
-            <input placeholder={textholder} disabled={disabled} id="form-input"/>
+            <label className="form-label">{name}</label>
```

Comment:
Acho que me expressei mal, precisa ter o id e htmlFor no label, apotando para um id do input, e o id do input precisa ser único
```suggestion
            <label className="form-label" htmlFor="{inputId}" >{name}</label>
```